MediaRecorder.bs

```diff
@@ -241,6 +249,32 @@ interface MediaRecorder : EventTarget {
     the value might be surpassed, not achieved, or only be achieved over a long
     period of time.</li>
 
+    <li>Let |videoKeyFrameIntervalDuration| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalDuration]]=] slot if defined, and let
+    |videoKeyFrameIntervalCount| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalCount]]=] slot if defined. The UA SHOULD constrain the
```

Comment:
I couldn't find any precedent for using _"if defined"_ to mean _"not `undefined`"_, so I'd like to avoid introducing that pattern here. Switching to `null` default values above should solve this. 

This should also simplify these assignments, which otherwise appear to be partially set:
```suggestion
    <li>Let |videoKeyFrameIntervalDuration| be
    |recorder|.[=[[VideoKeyFrameIntervalDuration]]=], and let
    |videoKeyFrameIntervalCount| be
    |recorder|.[=[[VideoKeyFrameIntervalCount]]=]. The UA SHOULD constrain the
```